models/reporting/commissioning/demographics/dim_person_demographics_basic.sql

```
@@ -89,16 +93,21 @@ and dict_gp.end_date is null
 left join (
-        select distinct pcn_code, borough
-        from {{ref('stg_reference_lookup_ncl_gp_practice')}} 
-) reg_bor_backup
-on gp_lu.pcn_code = reg_bor_backup.pcn_code
-
-left join {{ref('stg_reference_lookup_ncl_ncl_gp_practice_neighbourhood')}} nb_reg
+        -- ODS-derived borough/sub-ICB, one row per practice
+        select practice_code,
+            any_value(borough_registered) as borough_registered,
+            any_value(sub_icb_code)        as sub_icb_code,
+            any_value(sub_icb_name)        as sub_icb_name
```

SQLFluff (4.2.2)

[error] Select targets should be on a new line unless there is only one select target.

(LT09)